From: Jiri Pirko <jiri@mellanox.com>
Date: Mon, 25 Sep 2017 10:58:21 +0200
Subject: net: sched: introduce helper to identify gact pass action
Patch-mainline: v4.15-rc1
Git-commit: 3b8e9238a8d194e82f0202a5fb68a63686ebe420
References: bsc#1064802 bsc#1066129

Introduce a helper called is_tcf_gact_pass which could be used to
tell if the action is gact pass or not.

Signed-off-by: Jiri Pirko <jiri@mellanox.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 include/net/tc_act/tc_gact.h |    5 +++++
 1 file changed, 5 insertions(+)

--- a/include/net/tc_act/tc_gact.h
+++ b/include/net/tc_act/tc_gact.h
@@ -33,6 +33,11 @@ static inline bool __is_tcf_gact_act(con
 	return false;
 }
 
+static inline bool is_tcf_gact_ok(const struct tc_action *a)
+{
+	return __is_tcf_gact_act(a, TC_ACT_OK, false);
+}
+
 static inline bool is_tcf_gact_shot(const struct tc_action *a)
 {
 	return __is_tcf_gact_act(a, TC_ACT_SHOT, false);
